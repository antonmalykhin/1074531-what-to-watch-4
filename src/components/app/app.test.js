import React from 'react';
import renderer from 'react-test-renderer';
import App from './app.jsx';

const Film = {
  TITLE: `The Grand Budapest Hotel`,
  GENRE: `Drama`,
  RELEASE: 2014
};

const FILM_TITLES = [
  `Fantastic Beasts: The Crimes of Grindelwald`,
  `Bohemian Rhapsody`,
  `Macbeth`,
  `Aviator`,
  `We need to talk about Kevin`,
  `What We Do in the Shadows`,
  `Revenant`,
  `No Country for Old Men`
];

it(`Render App`, () => {
  const tree = renderer
    .create(<App
      title={Film.TITLE}
      genre = {Film.GENRE}
      release={Film.RELEASE}
      filmTitles={FILM_TITLES}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
