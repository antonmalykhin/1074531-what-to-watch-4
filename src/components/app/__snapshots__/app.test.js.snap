// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render App 1`] = `
<div
  className="loader"
>
  <div
    className="loader__ellipsis"
  >
    <div />
    <div />
    <div />
    <div />
  </div>
  <p
    className="loader__text"
  >
    Loading...
  </p>
</div>
`;
